### PR TITLE
Bugfix: Move program.cs into namespace

### PR DESCRIPTION
**Pull request by SvyatoS1**

### CODE REVIEW

File: Streetcode/Streetcode.XUnitTest/Streetcode.XUnitTest.csproj
Comment (MrVenik) [inline Streetcode/Streetcode.XUnitTest/Streetcode.XUnitTest.csproj:1]: Remove unnecessary changes in project file

File: Streetcode/DbUpdate/Program.cs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ﻿using DbUp;
 using Microsoft.Extensions.Configuration;
 
+namespace DbUpdate;
+
 public class Program
 {
     static int Main(string[] args)
```
Comment (MrVenik) [inline Streetcode/DbUpdate/Program.cs:8]: Add accessibility modifier

**Review by MrVenik — CHANGES_REQUESTED** [review]
- Remove unnecessary changes in project file
- Add accessibility modifier

**Review by HlibBondarev — APPROVED** [review]
Well done!

**Review by MrVenik — APPROVED** [review]
